CONTRIBUTING.md

```diff
@@ -1,118 +1,120 @@
-cannot be satisfied by the subscription IDs already in the repository, add
-the subscription ID to the notebook and email <devrel@planet.com> to notify us
-of the change. (**NOTE**: this is likely to change to a request when demo 
-permissions for the analytics feed are established).
-
-## Notebook Validation
-
-To enable validation of the Docker image, every notebook should run successfully
-when run from the command line. For notebooks where that just is not possible, 
-the notebooks can be excluded from automated running by adding its path to 
-[tests/skip_notebooks](tests/skip_notebooks). Excluding a notebook from automated running
-means that it is excluded from Docker Image validation. **If a notebook is
-skipped, it will not be guaranteed to be supported by the Docker image.**
-
-# Docker Image
-
-## Validation
-
-**Every time** the Docker image is changed, at the very least ensure that the 
-python packages still import without error by running the
-[imports_test notebook](dev/imports_test.ipynb).
-
-It is also strongly recommended that you ensure the Docker image can run all
-of the notebooks in the repository. This can be accomplished by automatically
-running all of the notebooks using the supplied test script. To run the test script,
-run the notebook in interactive mode, achieved by adding `/bin/bash` to 
-the container run command, e.g.
-```bash
-docker run -it --rm -p 8888:8888 \
-    -v $PWD:/home/jovyan/work \
-    -e PL_API_KEY='[YOUR-API-KEY]' \
-    planet-notebooks /bin/bash
+Contributions are welcome to this repository. Please open a PR if you would like to contribute a notebook and we will review. Alternatively, if you have a request or an idea, you can [open an issue](https://github.com/planetlabs/notebooks/issues/new/choose).
+
+## Notebook Style
+
+If you are contributing a notebook, we've gathered these style guide requirements.
+
+### Filemames
+
+Notebook filenames should not have spaces. Please use underscores with `snake_case` instead for all files in the repository.
+
+### HTML and Markdown
+
+For maximum portability, accessibility, and ease of use, notebooks in this repository should not use a custom style or theme. Please stick to standard markdown features so that it works across all environments.
+
+### Dependencies
+
+When a new notebook has a dependency that is not yet supported by the Docker image, please add the dependency to the [Docker setup requirements](planet-notebook-docker/requirements.txt).
+
+### Ordering Notebooks
+
+If you are working on a multi-part guide that has several notebooks, prefix the notebooks with `1_`, `2_`, `3_`, and so on to indicate the order.
+
+### Optimize Cell Outputs for Viewing in the Browser
+
+If you have a cell that prints a very large json repsonse, consider alternative ways to view the data. Could it be viewed in a table or on a map? Very large json repsonses make notebooks difficult to read in GitHub. 
```

Comment:
You can probably leave off the "in GitHub". Large jsons make notebooks hard to read in most viewers.